server/src/test/java/org/opensearch/common/util/LongRHHashTests.java

```diff
@@ -0,0 +1,48 @@
+/*
+ * SPDX-License-Identifier: Apache-2.0
+ *
+ * The OpenSearch Contributors require contributions made to
+ * this file be licensed under the Apache-2.0 license or a
+ * compatible open source license.
+ */
+
+package org.opensearch.common.util;
+
+import org.opensearch.test.OpenSearchTestCase;
+
+import java.util.Map;
+import java.util.TreeMap;
+
+public class LongRHHashTests extends OpenSearchTestCase {
+
+    public void testFuzzy() {
```

Comment:
can we add a test to verify the number of keys in the hash table for a static data set?

Reply:
Ack.